integration-tests/mongodb-panache/src/main/java/io/quarkus/it/mongodb/panache/bugs/BugResource.java

```diff
@@ -27,6 +33,10 @@
     @Inject
     Bug13301Repository bug13301Repository;
 
+    @Inject
+    @Named("cl2")
```

Comment:
what is "cl2" ?

Reply:
This is the name of the Mongo client to inject, there is two Mongo client in the integration test (pointing to the same cluster). I think `<default>` should also work.